sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -957,10 +957,10 @@ def _verify_interpreter_version_is_supported(pipeline_options):
   if sys.version_info[0] == 2:
     return
 
-  if sys.version_info[0] == 3 and sys.version_info[1] == 5:
-    if sys.version_info[2] < 3:
+  if sys.version_info[0:2] in [(3, 5), (3, 6)]:
+    if sys.version_info[0:3] < (3, 5, 3):
```

Comment:
Can we remove this warning now?

IIUC, According to @tvalentyn 3.5.0 now works.

Reply:
I recall this was a patch for typehints which may be what this was referring to.  @tvalentyn should we keep this recommendation regardless?

Reply:
I will send out a separate change if @tvalentyn believes this is no longer necessary.

Reply:
After typehints patch, we are not aware of issues why 3.5.0 might not work. It's a good question whether we want to recommend the user to use the same exact version, or higher version, as we install on Dataflow workers, for example: >= 2.7.13, >= 3.5.6, etc.